pyproject.toml

```diff
@@ -200,7 +200,7 @@ ignore = [
 # NOTE: PyTest options moved to dedicated `pytest.ini`
 
 [tool.airbyte_ci]
-python_versions = ["3.10", "3.11"]
+python_versions = ["3.10", "3.11", "3.12", "3.13"]
```

Comment:
This whole section is stale, as airbyte-ci is deprecated. Delete the entire `tool.airbyte_ci` section.